ts/utils/hooks/useOnFocus.tsx

```diff
@@ -13,17 +13,16 @@ export const useActionOnFocus = (
   dontExecuteBefore: Millisecond | undefined = undefined
 ) => {
   const [lastUpdate, setLastUpdate] = useState<Date | undefined>(undefined);
+  const isFocused = useIsFocused();
 
-  const navigation = useNavigation();
-  const isFocused = navigation.isFocused();
   useEffect(() => {
```

Comment:
Should we add
`// eslint-disable-next-line react-hooks/exhaustive-deps`
to the dependency array of this hook, in order to silence one of the 7 warning the linter produces?

Reply:
I think it is better to rewrite this hook in another PR since it is used elsewhere in the application